packages/core-components/src/components/TopBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+export interface TopBarItemType {
+  className: string;
+  linkContact1: string;
+  linkContact2: string;
+  phoneNumber1: string;
+  phoneNumber2: string;
+  nameContact1: string;
+  nameContact2: string;
+}
+
+export interface TopBarProps {
+  topBarItems: TopBarItemType[];
+  brandIcon: React.ReactNode;
+  phoneIcon: React.ReactNode;
+  languageButton: React.ReactNode;
+}
+
+const TopBar: React.FC<TopBarProps> = ({
+  topBarItems,
+  phoneIcon,
+  languageButton,
+}) => {
+  const itemsProps = topBarItems[0];
+  return (
+    <div
+      className={`${itemsProps.className} container sm:bg-white md:bg-primary flex flex-row items-center w-screen`}
```

Comment:
why do the container has its bg white when at `sm` then primary color as `md` ? where do we have this?